core/src/main/java/org/apache/accumulo/core/client/admin/InstanceOperations.java

```diff
@@ -57,19 +56,80 @@ void setProperty(final String property, final String value)
    * property overrides in ZooKeeper. Only properties which can be stored in ZooKeeper will be
    * accepted.
    *
+   * <P>
```

Comment:
Please use lowercase for html tags to comply with HTML5 conventions and W3C recommendations (and are entirely disallowed in xhtml). I keep fixing these when they get added as uppercase, but they keep getting added using uppercase again, and they are really hard to grep for or use automated tooling to check, because generics also use the less than and greater than symbols, and html is notoriously difficult to parse properly. Parsing inside a javadoc comment is even trickier.